### ISSUE TITLE
No instructions on setting up jumpbox? 

### ISSUE THREAD

**Issue by MattDevOps** [kelseyhightower/kubernetes-the-hard-way#782]
Is it implied that we need to look it up? It says to ssh root@jumpbox but I of course get 'no address associated with hostname'

**Comment by dcoy** [comment]
That's correct -- it's implied that you'll know the hostname or IP of your server that you'll be executing commands from. If you want a 1:1 match, you can map the `jumpbox` hostname to the server you'll be using:

```shell
Host jumpbox
  HostName <your-hostname-here>
  StrictHostKeyChecking yes
  PreferredAuthentications publickey
  IdentityFile ~/path/to/ssh-key
```

After doing so, executing `ssh root@jumpbox` will allow you to connect.

If you ___haven't___ setup a server for this, then it will be your local workstation. This is where the following applies:

>While a dedicated machine is being used to ensure consistency, these commands can also be run from just about any machine including your personal workstation running macOS or Linux

I hope this helps!
